android/app/src/main/AndroidManifest.xml

```diff
@@ -37,12 +37,14 @@
   <uses-permission android:name="android.permission.WAKE_LOCK"/>
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_LOCATION"/>
+
+  <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS"/>
   <!--
     Android 13 (API level 33) and higher supports a runtime permission for sending non-exempt (including Foreground
     Services (FGS)) notifications from an app.
   //-->
   <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
-
+  <uses-permission android:name="android.permission.ACCESS_NOTIFICATION_POLICY" />
```

Comment:
using this permission we can show notifications on lock screen as well
Its implementation was already there but since this permission was not there thats why notifications for navigations and trace path were not shown on lockscreen.

Reply:
OSMand, for example, does not declare this permission. Is it really needed?

Reply:
> What breaks if we remove this permission? 

In my xiaomi device when i use this permission it shows notifications of OM on lockscreen and without this permission it do not shows anything. 

But i tested on realme and motorola devices and nothing actually happens with and without this permission. Notification do not appears on lock screen in any case.

I feel this permission can be removed.